frontend/scripts/utils/getURLFromParams.js

```diff
@@ -45,6 +51,19 @@ function getURLForV2(endpoint, params = {}) {
   }, `${API_V2_URL}${endpoint}?`);
 }
 
+function getURLForContent(endpoint, params = {}) {
```

Comment:
This file needs to be refactored as there is now 4 times more or less the same logic to build endpoints URLs.

Reply:
done. Pretty soon we'll have a single API, so we'll simplify it even more